js/colors/rock-colors.ts

```diff
@@ -92,6 +92,10 @@ export const getRockCanvasPattern = (ctx: CanvasRenderingContext2D, rock: Patter
   return canvasPattern || "#ccc";
 };
 
+export const getRockCanvasPatternGivenNormalizedAge = (ctx: CanvasRenderingContext2D, rock: PatternName, normalizedAge: number) => {
```

Comment:
Could also use:
```
export const getRockCanvasPatternGivenNormalizedAge = (ctx: CanvasRenderingContext2D, rock: PatternName, normalizedAge = 0) => {
```
and then clients would not have to provide the `|| 0`.

Reply:
This was actually intentional, so if you try to get a rock pattern and don't think about age / normalized age, you don't try to use this function (and use `getRockCanvasPattern` instead). :wink: `|| 0` is never actually triggered in real life I believe, it's just a result of a loose interface.